pkgs/top-level/all-packages.nix

```diff
@@ -972,8 +972,6 @@ with pkgs;
 
   airspyhf = callPackage ../applications/radio/airspyhf { };
 
-  airtame = callPackage ../applications/misc/airtame { };
```

Comment:
Can you add an alias on `pkgs/top-level/aliases.nix` with an explanation of its removal?

Reply:
Sure, is `airtame has been removed due to being unmaintained` okay?